app/routes.py

```diff
@@ -1,2 +1,304 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, make_response, request
+from app import db
+import requests
+import os
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+tasks_bp = Blueprint("tasks", __name__, url_prefix = "/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
 
+
+# THE LINK TO TEST IN HEROKU IS https://task-list-project2.herokuapp.com/tasks
+
+
+@tasks_bp.route("",methods=["POST", "GET"])
+def handle_tasks():
+    
+    # get response body inJSON
+    if request.method == "POST":
+        request_body = request.get_json()
+        # return message if something ismissing
+        if 'title' not in request_body or 'description' not in request_body or 'completed_at' not in request_body:
+            response_body = {
+                'details': 'Invalid data'
+            }
+            return make_response(jsonify(response_body), 400)
```

Comment:
The `make_response()` and `jsonify()` here are redundant. Both are used to return a specific data type.

Our curriculum tends to focus on using `make_response()` for generating response data. `make_response` is a Flask function and returns `HTML`. Using this function will pass the tests for this project, however, if a client (web app) expects to receive JSON, the client would need to do extra work to access the dictionary. Or in this API, we can provide additional logic to change the headers for `make_response` to return JSON instead.

We can avoid all this extra work by either returning a dictionary (Flask automatically turns dictionaries into JSON) or by using `jsonify()`. 

So in this case you can omit `make_response` and `jsonify()` when returning a dictionary.

```suggestion
        return response_body, 400
```